ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Combinator/Serialisation/Common.hs

```diff
@@ -210,6 +218,13 @@ isHardForkNodeToClientEnabled :: HardForkNodeToClientVersion xs -> Bool
 isHardForkNodeToClientEnabled HardForkNodeToClientEnabled {} = True
 isHardForkNodeToClientEnabled _                              = False
 
+
+-- absolutely horrible. should be HasLatestStableGenTxIdEra probably.
+-- especially nasty because it's a K () rather than an equivalent (but
+-- non-existent) 'Void :: (Type -> Type) -> Type'
```

Comment:
> it's a `K ()` rather than an equivalent (but non-existent) `Void :: (Type -> Type) -> Type`

That's `Proxy`, right? It's argument is poly-kinded, so can be instantiated with `Type -> Type`. However, `K ()` also sounds fine.

Reply:
yeah, that's a good point actually! fixed 👍 

(been working a bunch with the `barbies` library recently, and used to having `Void` from there lying around lol)